src/components/gltf-model-plus.js

```diff
@@ -569,7 +662,7 @@ AFRAME.registerComponent("gltf-model-plus", {
       // If we had inflated something already before, clean that up
       this.disposeLastInflatedEl();
 
-      this.model = gltf.scene || gltf.scenes[0];
+      this.model = gltf.scene;
```

Comment:
Is this guaranteed to be set now? I think the reason we had it originally is that not GLTFs are not required to have a default scene (though almost all I have encountered in practice do).

Reply:
[Yes, it's guaranteed in the latest `GLTFLoader`](https://github.com/mrdoob/three.js/blob/r128/examples/jsm/loaders/GLTFLoader.js#L2042). IIRC the official loader didn't do that before so I think you encountered the problem with the old loader.